pandas/core/computation/eval.py

```diff
@@ -207,13 +206,10 @@ def eval(expr, parser='pandas', engine=None, truediv=True,
         scope. Most users will **not** need to change this parameter.
     target : a target object for assignment, optional, default is None
         essentially this is a passed in resolver
-    inplace : bool, default True
-        If expression mutates, whether to modify object inplace or return
-        copy with mutation.
-
-        WARNING: inplace=None currently falls back to to True, but
-        in a future version, will default to False.  Use inplace=True
-        explicitly rather than relying on the default.
+    inplace : bool, default False
+        If the expression mutates, whether to modify the expression inplace
```

Comment:
I think adding an example might clarify it further eg: "If the expression mutates (e.g. creates a new column), ..."